tests/integration/test_rest_handler_handler.py

```diff
@@ -50,3 +53,44 @@ def test_inputs_api_call():
         response.raise_for_status()
     except requests.exceptions.HTTPError as e:
         assert False, f"Exception {e}"
+
+
+def test_400_api_call():
+    expected_msg = """<msg type="ERROR">Unexpected error "&lt;class 'splunktaucclib.rest_handler.error.RestError'&gt;" 
+from python handler: "REST Error [400]: Bad Request -- HTTP 400 Bad Request -- 
+b'{"messages":[{"type":"ERROR","text":"Object id=demo://test_input cannot be deleted in config=inputs."}]}'". 
+See splunkd.log/python.log for more details.</msg>"""
+    try:
```

Comment:
Can we remove `try / except` and `response.raise_for_status` and instead check for `response.status_code == 500`? Same below.

Reply:
I don't want to have any logic in the tests, we should aim to have those tests as simple as we can.